xmodule/course_block.py

```diff
@@ -296,6 +296,27 @@ def get_available_providers():  # lint-amnesty, pylint: disable=missing-function
     return available_providers
 
 
+def get_requires_escalation_email_providers():  # lint-amnesty, pylint: disable=missing-function-docstring
+    proctoring_backend_settings = getattr(
+        settings,
+        'PROCTORING_BACKENDS',
+        {}
+    )
+
+    requires_escalation_email_providers = [
+        provider
+        for provider in proctoring_backend_settings
+        if provider != "DEFAULT"
+        and proctoring_backend_settings[provider].get(
+            "requires_escalation_email", False
+        )
+    ]
```

Comment:
openedx/envs/common.py defines PROCTORING_BACKENDS, so we can safely assume that it is always present on the `settings` object. No need to use `getattr`, which is error-prone and indirect.

```suggestion
    requires_escalation_email_providers = [
        provider
        for provider in settings.PROCTORING_BACKENDS
        if provider != "DEFAULT"
        and settings.PROCTORING_BACKENDS[provider].get(
            "requires_escalation_email", False
        )
    ]
```